controllers/user.js

```diff
@@ -167,10 +167,74 @@ class Users {
       username: user.username
     };
 
+
+    const getUserProfile = await User.findOne(
```

Comment:
This block of code is exactly the same with the one below. You will have to extract that into a function and call it in `login` and `register` user.

Reply:
ok

Reply:
I have made changes